apps/server/src/modules/oauth/controller/api-test/oauth-sso.api.spec.ts

```diff
@@ -329,7 +329,10 @@ describe('OAuth SSO Controller (API)', () => {
 					.set('Cookie', cookies)
 					.query(query)
 					.expect(302)
-					.expect('Location', `${clientUrl}/login?error=sso_auth_code_step`);
+					.expect(
+						'Location',
+						`${clientUrl}/migration/error?errorcode=OauthMigrationFailed&sourceSystem=undefined&targetSystem=undefined&sourceSchoolNumber=undefined&targetSchoolNumber=undefined`
```

Comment:
Can you create a test where the params are not undefined?

Reply:
done